pkg/services/authn/clients/anonymous.go

```diff
@@ -14,6 +15,8 @@ import (
 
 var _ authn.ContextAwareClient = new(Anonymous)
 
+const timeoutTag = 2 * time.Minute
```

Comment:
Question: why two minutes? That seems like a very long time for an operation that should take way less.

Reply:
giving leeway, I can put it down. What value would you recommend?